Assets/HoloToolkit/Input/Tests/Scripts/SimpleGridGenerator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class SimpleGridGenerator : MonoBehaviour
```

Comment:
I think even if the objects are placed directly in the scene we're still dynamically choosing which one to send the message too based on gaze. I believe this is a clear enough demonstration without needing to actually dynamically add the objects to the scene and removing this would make the pull request a lot simpler.